README.md

```diff
@@ -1,159 +1,170 @@
+
+> `./gradlew hubPullChanges` will overwrite any local changes you have made that were not pushed to the database using `./gradlew hubDeployUserArtifacts`
+
+### Deployment best practices and caveats
 
-This command will run in the background and automatically upload changes you make to artifacts in the project directory -- see ml-gradle documentation for details: [Watching for Module Changes](https://github.com/marklogic-community/ml-gradle/wiki/Watching-for-module-changes)
+If you happen to clear or delete all of your user data from the staging database, `data-hub-STAGING`, you will need to re-ingest the reference data by running `./gradlew mlLoadData`
 
-### Deployment best practices and caveats:
+This will restore the reference document contents found in the `referenceData/` directory into the collection required to run user-defined steps included with the project.
 
-If you happen to clear or delete all of your user data from the staging database, `data-hub-STAGING`, you will need to re-ingest the reference data via this command: 
+### Loading the SNOMED-CT Ontology
 
-- `./gradlew mlLoadData`
+If you want to load a SNOMED-CT Ontology into your HSK instance, you will need to download the ontology yourself as it requires a license for use and distribution.
 
-This will restore the refrerence document contents found in the `referenceData/` directory into the collection required to run user-defined steps included with the project.
\ No newline at end of file
+Once downloaded you will need to run the ZIP file through the [`snomed-owl-toolkit`](https://github.com/IHTSDO/snomed-owl-toolkit) and then run the resulting `ontology-<time-run>.owl` through [ROBOT](http://robot.obolibrary.org/) in order to transform the data into a format that will be understood by [MLCP](https://github.com/marklogic/marklogic-contentpump) for ingestion. Once transformed place the ingestable file at `./src/main/ml-data/ontologies/SNOMED-CT.ttl` and you will be able to run `./gradlew loadSnomedCTOntology` (or if you want to use all 3 ontologies you can run `./gradlew loadOntologies`).
+
+```sh
+# Example commands for transforming SNOMED-CT ontology into an ingestable format
+# from the downloaded ZIP file, as run from the project directory
+#
+# Convert the RF2 files contained in the ZIP file to Functional OWL syntax.
+# Example output filename: ~/downloads/ontology-2022-01-19_11-05-40.owl
+java -jar -xms4g snomed-owl-toolkit-3.0.3-executable.jar \
+     -rf2-to-owl ~/downloads/SnomedCT_PRODUCTION.zip
+# Convert from Functional OWL to TTL
+java -jar robot.jar convert \
+     --input ~/downloads/ontology-2022-01-19_11-05-40.owl \
+     --output ./src/main/ml-data/ontologies/SNOMED-CT.ttl
+```

```

Comment:
At the end, or in the "extending" section, refer people to the Cookbook for a list of features and how to modify functionality to use each project's actual data set and needs.